### DISCUSSION TITLE
Does doom-nvim have multi cursor function?

### DISCUSSION THREAD

**Discussion by dwchoo** [NTBBloodbath/doom-nvim#412] [Q&A]
Hi,
II wonder if doom-nvim has a feature like vim-visual-multi(https://github.com/mg979/vim-visual-multi).
And if anyone uses vim-visual-multi, the ctrl+n hotkey overlaps. I wonder how to use it.

**Comment by connorgmeehan** [comment]
Hey @dwchoo, it doesn't have builtin support for multi-cursor but that can be added by adding the following line to your config.lua
```lua
doom.use_package("mg979/vim-visual-multi")
```

**Reply by dwchoo** [reply]
Thank you for your answer. but 'vim-visual-multi' has same hot key with default configuration.
In the Doom-nvim, ctrl+n is 'jump to next diagnostic'. Do I have to change it?
